book/_config.yml

```diff
@@ -0,0 +1,55 @@
+# Book settings
+# Learn more at https://jupyterbook.org/customize/config.html
+
+title: CVX Short Course
+author: CVX
+logo: static/logo.jpeg
+only_build_toc_files: true
+
+execute:
+  execute_notebooks: force   #auto   # only execute if there is at least on output cell missing
+  timeout: 300  # 5 minutes timeout
+  allow_errors: true
+  exclude_patterns:
+    - _build
+    - .DS_Store
+    - "*.ipynb_checkpoints"
+    - "exercises/notebooks/*.ipynb"
+    - "hello/notebooks/*.ipynb"
+
+sphinx:
+  config:
+    html_js_files:
+    - https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js
+
+parse:
+  myst_enable_extensions:
+    - substitution
+    - linkify
+    - dollarmath
+    - amsmath
+
+
+# Information about where the book exists on the web
+# ATTENTION. Please correct the url back to cvxgrp
+repository:
+  url: https://github.com/tschm/cvx_short_course  # Online location of your book
```

Comment:
Should be `cvxgrp`

Reply:
Yes, this should be changed first thing after the merge... otherwise the book fails on my repo. Did you have a look at https://tschm.github.io/cvx_short_course/docs/index.html?